projects/helper/portedTokens.js

```diff
@@ -522,7 +522,7 @@ function transformOasisAddressBase(addr) {
     '0xdc19a122e268128b5ee20366299fc7b5b199c8e3': '0xdac17f958d2ee523a2206206994597c13d831ec7', // USDT wormhole
     '0x81ecac0d6be0550a00ff064a4f9dd2400585fe9c': '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', // USDC celer
     '0x94fbffe5698db6f54d6ca524dbe673a7729014be': '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',  // USDC
-    //'0x21c718c22d52d0f3a789b752d4c2fd5908a8a733': 'wrapped-rose',
+    '0x21c718c22d52d0f3a789b752d4c2fd5908a8a733': 'wrapped-rose',
```

Comment:
sorry, but I think you have fix the merge conflicts in this file in a bad manner, basically reverted a lot of changes that we have done., this will break a lot of our other adapters, can you please fix this file?

All your other changes look really good :)